RecoLocalCalo/Configuration/python/RecoLocalCalo_cff.py

```diff
@@ -20,7 +20,8 @@
 #
 # sequence CaloLocalReco and CaloGlobalReco
 #
-calolocalreco = cms.Sequence(ecalLocalRecoSequence+hcalLocalRecoSequence)
+calolocalrecoTask = cms.Task(ecalLocalRecoTask,hcalLocalRecoTask)
+calolocalreco = cms.Sequence(calolocalrecoTask)
 caloglobalreco = cms.Sequence(hcalGlobalRecoSequence)
```

Comment:
Why not also migrating this sequence?

Reply:
Indeed, this sequence is never used, it only includes ```hcalGlobalRecoSequence``` from ```RecoLocalCalo/Configuration/python/hcalGlobalReco_cff.py``` (which in its turn is only used for the ```globalreco_tracking``` reconstruction sequence) and it can be misleading here: I would suggest to just remove it